.gitpod.Dockerfile

```
@@ -1,7 +1,7 @@
 FROM gitpod/workspace-full:2023-01-16-03-31-28
 
 RUN mkdir -p ~/.local/bin
-RUN curl -L -o ~/.local/bin/soroban https://github.com/stellar/soroban-tools/releases/download/v0.9.1/soroban-cli-0.9.1-x86_64-unknown-linux-gnu
+RUN curl -L https://github.com/stellar/soroban-tools/releases/download/v0.9.1/soroban-cli-0.9.1-x86_64-unknown-linux-gnu.tar.gz | tar xz -C ~/.local/bin soroban
```

space over slash?

```suggestion
RUN curl -L https://github.com/stellar/soroban-tools/releases/download/v0.9.1/soroban-cli-0.9.1-x86_64-unknown-linux-gnu.tar.gz | tar xz -C ~/.local/bin/soroban
```

could be wrong, I'm unfamiliar with the `-C` param

---

It means "Change into this directory before moving on" or some such thing... Haven't used it before, just kinda copied what else was happening in the dockerfile. Whatever it means, the executable does end up at `~/.local/bin/soroban` 🤷🏻

---

good enough for me!